font-awesome-4.7.0/less/animated.less

```diff
@@ -0,0 +1,34 @@
+// Animated Icons
```

Comment:
Why aren't you using CDN for fontawesome? you should use fontawesome CDN instead of keeping it locally.